src/List/style.css

```diff
@@ -0,0 +1,44 @@
+@import 'antd/lib/list/style/index.css';
+@import 'antd/lib/skeleton/style/index.css';
```

Comment:
Why do you need to import skeleton style? It does not exist in css.js. If you need this for the story, you can move this import to `index.stories.tsx`